### PR TITLE
Configure docker extension to use `node.js` path modules

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

### This PR fixes:
There was error coming while building because the @layer5/sistent package or any project that is attempting to use the Node.js path module. Since Webpack 5 does not include polyfills for Node.js core modules by default, this causes the build to fail when trying to resolve path.
Now we can use sistent package easily.
<hr/>

### Why used react script rewired:
react-app-rewired allows you to customize React App configurations (like Webpack) without ejecting.

❌**Without react-app-rewired:**
 Customization requires ejecting, which exposes configuration files but makes maintenance harder.

✅**With react-app-rewired:** 
You can modify configurations while staying within the managed CRA setup, keeping upgrades easier and your project simpler to maintain.

> [!IMPORTANT]
> This is do not merge because i have to fix styles of catalog card properly from sistent and release then we can move forward.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/docker-extension/ui/config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const webpack = require('webpack');
+module.exports = function override(config, env) {
```
Comment (humblenginr) [inline install/docker-extension/ui/config-overrides.js:2]: @sudhanshutech why don't we use a webpack plugin for this - https://www.npmjs.com/package/node-polyfill-webpack-plugin

**Comment by humblenginr** [comment]
Thanks @sudhanshutech . `react-app-rewired` makes sense ✅

File: install/docker-extension/ui/config-overrides.js (hunk 1 above)
Comment (sudhanshutech) [reply install/docker-extension/ui/config-overrides.js:2]: This plugin is not needed when we using the react app rewired so we can easily configure the webpack configurations.
And when i gone through there some discussion it not solving few things completely.

File: install/docker-extension/ui/package-lock.json
Comment (humblenginr) [inline install/docker-extension/ui/package-lock.json:1]: should we include package-lock in this pr?

Comment (sudhanshutech) [reply install/docker-extension/ui/package-lock.json:1]: yes it would be needed😅

**Review by leecalcote — APPROVED** [review]
LGTM

**Comment by sudhanshutech** [comment]
@humblenginr please give a final look and let me know so we can merge and the filp card will be available in  next release. I have to little work in sistent for that . But here everything is working fine so we don't have to wait.

**Review by humblenginr — APPROVED** [review]
LGTM 🎉

**Pull request merged**